xbmc/utils/DisplayInfo.cpp

```diff
@@ -28,6 +29,9 @@ void CDisplayInfo::DiInfoDeleter::operator()(di_info* p)
 std::unique_ptr<CDisplayInfo> CDisplayInfo::Create(const std::vector<uint8_t>& edid)
 {
   auto info = std::unique_ptr<CDisplayInfo>(new CDisplayInfo(edid));
+  if (!info)
+    return {};
+
```

Comment:
This shouldn't be necessary, new will never return `nullptr` if not called with the `std::nothrow_t` tag.